src/HotChocolate/ApolloFederation/src/ApolloFederation/ExternalAttribute.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Reflection;
+using HotChocolate.ApolloFederation.Extensions;
+using HotChocolate.Types;
+using HotChocolate.Types.Descriptors;
+
+namespace HotChocolate.ApolloFederation
+{
+    /// <summary>
+    /// The @external directive is used to mark a field as owned by another service.
+    /// This allows service A to use fields from service B while also knowing at runtime
+    /// the types of that field.
+    ///
+    /// <example>
+    /// # extended from the Users service
+    /// extend type User @key(fields: "email") {
+    ///   email: String @external
+    ///   reviews: [Review]
+    /// }
+    /// </example>
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Method)]
+    public sealed class ExternalAttribute : DescriptorAttribute
```

Comment:
If we do not have an extension case for an attribute it has to be sealed... that is why I changed it here.

Reply:
Acknowledged